dev/benchmarks/platform_views_layout/android/project-google_mobile_ads.lockfile

```diff
@@ -0,0 +1,126 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
these 2 lockfiles are generated because admob dep is added. @gmackall

Reply:
If it was generated by the script then it LGTM. I tried building with these 2 files as well and didn't hit any issues, so they should be safe to include